packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/PageNames/AccountName/AccountDetails.tsx

```diff
@@ -72,7 +72,7 @@ export const AccountDetails = ({ selectedAccount }: AccountDetailsProps) => {
                 </CryptoBalance>
 
                 <span>
-                    ~<FiatValue amount={formattedBalance} symbol={symbol} />
+                    ≈ <FiatValue amount={formattedBalance} symbol={symbol} />
```

Comment:
nit:
not related by the gap is too big
![Screenshot 2024-04-18 at 10 50 19](https://github.com/trezor/trezor-suite/assets/33235762/fc6f1a9f-9d5b-4241-be24-a0396bcc9cab)

Reply:
e4991bc3e28cd6efa3cd43aa10f8a7a829b736b8